{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2019-25014"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "A NULL pointer dereference was found in pkg/proxy/envoy/v2/debug.go getResourceVersion in Istio pilot before 1.5.0-alpha.0. If a particular HTTP GET request is made to the pilot API endpoint, it is possible to cause the Go runtime to panic (resulting in a denial of service to the istio-pilot application)."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1919066",
          "refsource": "MISC",
          "tags": [],
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1919066"
        },
        {
          "name": "https://github.com/istio/istio/compare/1.4.2...1.5.0-alpha.0",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/istio/istio/compare/1.4.2...1.5.0-alpha.0"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2021-01-29T12:57Z",
  "publishedDate": "2021-01-29T06:15Z"
}